src/esp/metadata/MetadataMediator.h

```diff
@@ -30,9 +30,8 @@ namespace metadata {
 class MetadataMediator {
  public:
   MetadataMediator(const sim::SimulatorConfiguration& cfg);
-  MetadataMediator(const std::string& _activeSceneDataset = "default",
-                   const std::string& _physicsManagerAttributesPath =
-                       ESP_DEFAULT_PHYSICS_CONFIG_REL_PATH);
+
+  MetadataMediator() : MetadataMediator(sim::SimulatorConfiguration{}) {}
```

Comment:
This will have the created MM be based on a default SimulatorConfiguration. This is to unify handling of MM both from Simulator and if instantiated without an existing sim.

TODO : one caveat to this could be that if someone wanted to cycle through sim close/reconfigure whilst retaining an MM, they would need to remember to call reconfigure with MM's simulator configuration, or else MM's will be replaced with whatever config they use.